front/components/authPage/LinkExpired.vue

```diff
@@ -20,8 +20,8 @@
           <img width="325" src="@/assets/images/bird-illustration.png" alt="" />
         </div>
         <div v-if="!hideSend" style="margin-top: -70px;">
-          <a href="#" @click.prevent="send2ndTime">
-            <link-text active="active" text="Resend link"></link-text>
+          <a href="#" class="link-highlight" @click.prevent="send2ndTime">
```

Comment:
haha this. 😂 

not a change, already has a href=# @click.prevent, so not evaluating that part of this for the PR.